frontend/packages/ceph-storage-plugin/src/queries/ceph-queries.ts

```diff
@@ -213,3 +218,14 @@ export const getPVCUsedCapacityQuery = (pvcName: string): string =>
 export const osdDiskInfoMetric = _.template(
   `ceph_disk_occupation{exported_instance=~'<%= nodeName %>'}`,
 );
+
+export const getPoolQuery = (poolNames: string[], queryName: string) => {
+  const names = poolNames.join('|');
+  const queries = {
+    [StorageDashboardQuery.POOL_RAW_CAPACITY_USED]: `ceph_pool_stored_raw * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${names}'}`,
```

Comment:
```suggestion
    [StorageDashboardQuery.POOL_RAW_CAPACITY_USED]: `ceph_pool_used_bytes * on (pool_id) group_left(name)ceph_pool_metadata{name=~'${names}'}`,
```

ceph_pool_stored_raw - proves the used info after replication
ceph_pool_used_bytes - proves the used info after replication and compression

Reply:
Thanks for the explanation Anmol. It completely makesense to use ceph_pool_used_bytes instead of ceph_pool_stored_raw. i will change this.